fuel-core-interfaces/src/db.rs

```diff
@@ -140,17 +144,19 @@ pub mod helpers {
 
     #[derive(Clone, Debug)]
     pub struct Data {
+        /// Used for Storage<Address, (u64, Option<Address>)>
+        pub validators: HashMap<Address, (ValidatorStake, Option<Address>)>,
```

Comment:
an address mapping to an address isn't obvious without external context. consider adding some documentation here explaining what is what